crates/teloxide-core/src/types/message.rs

```diff
@@ -393,14 +411,15 @@ pub struct MediaDocument {
 
     /// For messages with a caption, special entities like usernames, URLs,
     /// bot commands, etc. that appear in the caption.
-    #[serde(default)]
+    #[serde(skip_serializing_if = "Vec::is_empty", default = "Vec::new")]
```

Comment:
BTW you don't need `= "Vec::new"`, since `Vec` [implements](https://doc.rust-lang.org/std/vec/struct.Vec.html#impl-Default-for-Vec%3CT%2C%20Global%3E) `Default`. If you could replace existing mentions of `default = "Vec::new
"` with `default` that would be nice.

Reply:
Agree, fixed

Reply:
@WaffleLapkin, did you mean:

```rust
#[serde(skip_serializing_if = "Vec::is_empty", default)]
```

?

Reply:
Yes

Reply:
@Olegt0rr you still did not address this comment, there are still `default = "Vec::new"` in the `message.rs`

Reply:
@Hirrolot I think this still wasn't addressed

Reply:
Just removed redundant "Vec::new"